src/ui/components/Account/Library.tsx

```diff
@@ -9,8 +9,7 @@ function Header() {
   return (
     <div id="header">
       <div className="header-left">
-        <div className="logo" />
-        <div className="title-label">Replay</div>
+        <div className="title-label">Your Replay library</div>
```

Comment:
do we need Replay? `Your library` might be punchier...

Reply:
I could see that. Will remove, thanks!

Reply:
Ok, change is in